src/main/java/racingcar/model/Cars.java

```diff
@@ -1,26 +1,33 @@
 package racingcar.model;
 
-import racingcar.RandomManager;
+import racingcar.MoveStrategy;
 
+import java.util.ArrayList;
 import java.util.List;
 import java.util.NoSuchElementException;
 import java.util.stream.Collectors;
 
 public class Cars {
-    private static final int MOVE_CONDITION = 3;
+    private static final int INIT_CAR_DISTANCE = 0;
 
     private List<Car> cars;
 
     public Cars(List<Car> cars) {
         this.cars = cars;
     }
 
-    public void moveCars() {
-        this.cars.forEach(car -> car.move(() -> isMovingForward(RandomManager.getInstance().getRandomValue())));
+    public static Cars of(List<String> names) {
+        List<Car> cars = new ArrayList<>(names.size());
+
+        for (String name : names) {
+            cars.add(new Car(name, INIT_CAR_DISTANCE));
+        }
+
+        return new Cars(cars);
     }
 
-    private boolean isMovingForward(int value) {
-        return value > MOVE_CONDITION;
+    public void moveCars(MoveStrategy strategy) {
```

Comment:
```suggestion
    public void move(MoveStrategy strategy) {
```
cars는 중복적인 의미가 있어보여요 move로도 충분할 것 같습니다 😄